### ISSUE TITLE
SecurityException not caught 

### ISSUE THREAD

**Issue by eos1d3** [jhy/jsoup#1119]
Using normal Jsoup 1.10.3 with request below:

```
try {
	Response response = Jsoup.connect(url)
		.header("accept-language", "en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4")
		.header("upgrade-insecure-requests", "1")
		.userAgent("Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:57.0) Gecko/20100101 Firefox/57.0")
		.method(Method.GET)
		.execute();
} catch (Exception e) {
   .....
}
```

It never had problem until today. There are exceptions which are NOT caught. Please note that the `try-catch` above should catch everything. But I still see these error messages and the program was terminated by it.

```
Exception in thread "Thread-1" java.lang.ExceptionInInitializerError
        at javax.crypto.JceSecurityManager.<clinit>(JceSecurityManager.java:65)
        at javax.crypto.Cipher.getConfiguredPermission(Cipher.java:2586)
        at javax.crypto.Cipher.getMaxAllowedKeyLength(Cipher.java:2610)
        at sun.security.ssl.CipherSuite$BulkCipher.isUnlimited(Unknown Source)
        at sun.security.ssl.CipherSuite$BulkCipher.<init>(Unknown Source)
        at sun.security.ssl.CipherSuite.<clinit>(Unknown Source)
        at sun.security.ssl.SSLContextImpl.getApplicableCipherSuiteList(Unknown Source)
        at sun.security.ssl.SSLContextImpl.access$100(Unknown Source)
        at sun.security.ssl.SSLContextImpl$AbstractTLSContext.<clinit>(Unknown Source)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Unknown Source)
        at java.security.Provider$Service.getImplClass(Unknown Source)
        at java.security.Provider$Service.newInstance(Unknown Source)
        at sun.security.jca.GetInstance.getInstance(Unknown Source)
        at sun.security.jca.GetInstance.getInstance(Unknown Source)
        at javax.net.ssl.SSLContext.getInstance(Unknown Source)
        at javax.net.ssl.SSLContext.getDefault(Unknown Source)
        at javax.net.ssl.SSLSocketFactory.getDefault(Unknown Source)
        at javax.net.ssl.HttpsURLConnection.getDefaultSSLSocketFactory(Unknown Source)
        at javax.net.ssl.HttpsURLConnection.<init>(Unknown Source)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.<init>(Unknown Source)
        at sun.net.www.protocol.https.Handler.openConnection(Unknown Source)
        at sun.net.www.protocol.https.Handler.openConnection(Unknown Source)
        at java.net.URL.openConnection(Unknown Source)
        at org.jsoup.helper.HttpConnection$Response.createConnection(HttpConnection.java:777)
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:649)
        at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:629)
        at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:261)
        at LiveRecorder.VideoRecorder.parseDashPlayBack(VideoRecorder.java:79)
        at LiveRecorder.VideoRecorder.run(VideoRecorder.java:193)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.SecurityException: Can not initialize cryptographic mechanism
        at javax.crypto.JceSecurity.<clinit>(JceSecurity.java:93)
        ... 31 more
Caused by: java.lang.SecurityException: Cannot locate policy or framework files!
        at javax.crypto.JceSecurity.setupJurisdictionPolicies(JceSecurity.java:316)
        at javax.crypto.JceSecurity.access$000(JceSecurity.java:50)
        at javax.crypto.JceSecurity$1.run(JceSecurity.java:85)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.crypto.JceSecurity.<clinit>(JceSecurity.java:82)
        ... 31 more
```

**Comment by noties** [comment]
`ExceptionInInitializerError` is a sibling of `Throwable` not an `Exception`, so changing to `catch (Throwable t)` will fix your issue

**Comment by krystiangorecki** [comment]
The "exception" in your stacktrace is _java.lang.ExceptionInInitializerError_ which is a subclass of _java.lang.Error_ and [documentation of Error ](https://docs.oracle.com/javase/7/docs/api/java/lang/Error.html)states:

> An Error is a subclass of Throwable that indicates serious problems that a reasonable application *should not try to catch*. Most such errors are abnormal conditions. 

The cause of this error is `java.lang.SecurityException: Cannot locate policy or framework files!` and you should try to solve this problem first. It doesn't have anything to do with Jsoup.

**Comment by eos1d3** [comment]
This exception is weird. The code run for a year without any issue. I do not think there is really SecurityException. 

After the error, I run it again and did not see any problem. But I saw Windows prompted for asking to allow Firewall access for the java application during my test. Can this be the reason? But before I clicked yes, the test had been running without any issue.

I did upgrade Java runtime to the latest version. Can this cause all the problem?
